sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -91,32 +95,88 @@ credential.update("{new_api_key}");
+
+### Recognizing values from forms
+Using the `FormRecognizerClient`, you can use your own trained models to recognize field values and locations, as well as
+table data, from forms of the type you trained your models on. The output of models trained with and without labels
+differs as described below.
+
+#### Using models trained without labels
+Models trained without labels consider each form page to be a different form type. For example, if you train your
+model on 3-page forms, it will learn that these are three different types of forms. When you send a form to it for
+analysis, it will return a collection of three pages, where each page contains the field names, values, and locations,
+as well as table data, found on that page.
+
+#### Using models trained with labels
+Models trained with labels consider a form as a single unit. For example, if you train your model on 3-page forms
+with labels, it will learn to recognize field values from the locations you've labeled across all pages in the form.
+If you sent a document containing two forms to it for analysis, it would return a collection of two forms,
+where each form contains the field names, values, and locations, as well as table data, found in that form.
+Fields and tables have page numbers to identify the pages where they were found.
+
+### Managing Custom Models
+Using the `FormTrainingClient`, you can get, list, and delete the custom models you've trained.
+You can also view the count of models you've trained and the maximum number of models your subscription will
+allow you to store.
 
 ## Examples
-The following sections provide several code snippets covering some of the most common Form Recognizer tasks, including:
 
-### Form Recognizer Client
-Form Recognizer support both synchronous and asynchronous client creation by using
-`FormRecognizerClientBuilder`,
+The following section provides several code snippets covering some of the most common Form Recognizer tasks, including:
 
-<!-- embedme ./src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java#L39-L42 -->
-``` java
-FormRecognizerClient formRecognizerClient = new FormRecognizerClientBuilder()
-    .apiKey(new AzureKeyCredential("{api_key}"))
-    .endpoint("{endpoint}")
-    .buildClient();
-```
-<!-- embedme ./src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java#L49-L52 -->
-``` java
-FormRecognizerAsyncClient formRecognizerAsyncClient = new FormRecognizerClientBuilder()
-    .apiKey(new AzureKeyCredential("{api_key}"))
-    .endpoint("{endpoint}")
-    .buildAsyncClient();
-```
+* [Recognize Receipts](#recognize-receipts "Recognize receipts")
+* [Recognize Content](#recognize-content "Recognize Content")
+* [Recognize Forms Using a Custom Model](#recognize-forms-using-a-custom-model "Recognize Forms Using a Custom Model")
+* [Train a Model](#train-a-model "Train a model")
+* [Manage Your Models](#manage-your-models "Manage Your Models")
 
 ### Extract receipt information
```

Comment:
I think you should change all mentions of 'extract' to 'recognize', and get rid of all mentions of 'layout', except for at the beginninig where we tie it to the service's layout endpoint